torch/_jit_internal.py

```diff
@@ -903,6 +903,8 @@ def _qualified_name(obj):
     # Enum classes do not have `__name__` attr, instead they have `name`.
     elif isinstance(obj, enum.Enum):
         name = obj.name
+    elif hasattr(obj, '__str__'):
+        return obj.__str__()
```

Comment:
Could you add a test for this?

Reply:
sure, is there any specific place or form for such tests?